tools/osx-pkg/scripts/npm_preinstall.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+# TODO Can this be done inside the .pmdoc?
+# TODO Can we extract $PREFIX from the installer?
+
+set -e
+
+rm -rf /usr/local/lib/node_modules/npm
```

Comment:
It would be great if we could test it on a build machine. Don't want to mess with my local working machine all the time. ;)